README.md

```diff
@@ -1 +1,46 @@
-# whatsapp-groupadd
\ No newline at end of file
+# Whatsapp-groupadd
+
+Whatsapp-groupadd is a Python code which helps it users to add people to a specific whatsapp group without directly saving their numbers.
+
+## Installation
+

```

Comment:
Mention `requirements` too. 
It's good to have pre-requisites before Installation steps.